### ISSUE TITLE
Plugin disables f-movement in macros

### ISSUE THREAD

**Issue by alexveden** [jonatan-branting/better-n.nvim#6]
When I use default settings, they are re-mapping [FfTf] keys to better-n functions.

However, in case of recording macros, recorded 'f<char>' were not reproduced when macro is called.

Here is a workaround to disable keys in config:
```
          ["f"] = {cmdline = true},
          ["t"] = {cmdline = true},
          ["F"] = {cmdline = true},
          ["T"] = {cmdline = true},
```

p.s. Maybe it's worth to find another option to make clear how to disable `better-n` calls and propagating original nvim calls. I've only managed to figure this out after digging into code. 

Thanks

**Comment by jonatan-branting** [comment]
Hey, thanks for the issue!

I'll think about of to fix the "mandatory" default overrides, I agree it's a bit problematic.

Regarding f-movements not working, I'll look into this and see if it's something that's fixable.

**Comment by jonatan-branting** [comment]
This should be solved with #10, although you will need to modify your configuration to work with the new version. (Sorry about that!)
